spec/pipeline_spec.rb

```diff
@@ -233,6 +233,27 @@ def call
         expect { subject.execute(ArgumentError) }.to raise_error(ArgumentError)
       end
     end
+
+    context 'when a handler was configured not to halt the pipeline' do
+      subject do
+        NxtPipeline::Pipeline.new do |pipeline|
+          pipeline.constructor(:error_test) do |step, arg|
+            step.raisor.call(arg)
+          end
+
+          pipeline.step :error_test, raisor: -> (error) { raise error }
+          pipeline.step :error_test, peacemaker: -> (error) { 'Second step does not raise an error' }
```

Comment:
Would be nice to actually see that the second step was executed.

Reply:
True, finished it in a rush after having left the task. I improved the specs to explicitly check that.